.github/workflows/ci.yml

```diff
@@ -14,15 +14,17 @@ jobs:
           apt update
           apt install -y sudo
           sudo apt install -y git
-          sudo DEBIAN_FRONTEND=noninteractive apt install --yes npm
-      - name: Install commitlint
-        run: npm install
-      - name: Print versions
+          
+          sudo apt install --yes curl
+          # can't install ubuntu's default nodejs version because we would get this error:
+          # error @commitlint/config-conventional@17.1.0: The engine "node" is incompatible with this module. Expected version ">=v14". Got "12.22.9"
+          curl -sL https://deb.nodesource.com/setup_18.x | sudo -E bash -
```

Comment:
@realmarv by the error message above it sounds like we don't need to be in such a bleeding edge version, let's try 14.x instead

Reply:
It broke the CI https://github.com/realmarv/conventions/actions/runs/3928561523/jobs/6716315668

Reply:
then 16.x?

Reply:
Done

Reply:
But why is the reason?

Reply:
I already said it above; do you understand what "bleeding edge" means in terms of versions? it means too high, potentially unstable; lower versions have been tested more, so more stable

Reply:
Got it thanks